### DISCUSSION TITLE
All Godot versions in brew [MacOS]

### DISCUSSION THREAD

**Discussion by 0xIonRod** [godotengine/godot-proposals#4588] [Platforms]
Hi all!

I'm newbie in Godot. 
I'm using MacOS  and it's amazing that it's possible to install Godot using brew.

However there only one version in repo – currently stable 3.4.4
Not so long time ago 3.5-rc2 was released. I have to install it manually.

It would be amazing to make it possible to instal different versions of Godot using

`brew install -cask godot@[version]`

I'm happy to help with this task is someone could help me to with integration of the solution into Godot:
1. How MacOs builds are made now?
2. How brew package was created?
3. Once I'll make the solution who could help me to prepare it to be pushed to the Godot workflow?

**Comment by Calinou** [comment]
The [`godot` Homebrew formula](https://github.com/Homebrew/homebrew-cask/blob/HEAD/Casks/godot.rb) is not maintained by us, so this is something that should be discussed with the maintainers of the `godot` Homebrew formula. How do other popular Homebrew formulae handle multiple packaged versions (such as Python and Go)?

I personally don't have experience authoring Homebrew formulas, so I wouldn't know how to support writing a formula so it supports installing multiple versions.

**Reply by 0xIonRod** [reply]
Amazing, thank you so much for reply. 

I'll try to reach those guys.

To handle several versions you just have to type:
```
brew install python@3.8 – installs python3.8
brew install python@3.9 – installs python3.9

```
Pretty easy
